metricbeat/module/jolokia/_meta/config.yml

```diff
@@ -4,6 +4,21 @@
   hosts: ["localhost"]
   namespace: "metrics"
   path: "/jolokia/?ignoreErrors=true&canonicalNaming=false"
-  jmx.mapping:
-  jmx.application:
-  jmx.instance:
+  jmx.mappings:
```

Comment:
Thanks for these examples. Are these mbeans always available in Jolokia/JMX?

Reply:
They have been part of Oracle JVM since 1.6 

Most user tools assume them to be present. Though the JMX standard doesn't explicitly state them. 

I'm not sure about IBM JDK.